### PR TITLE
Feature/fm 697 dashboard v closed

### PR DESCRIPTION
**Pull request by ramchurnk**

### CODE REVIEW

File: app/controllers/facilities_management/beta/procurements_controller.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ def index
         @searches = current_user.procurements.where(aasm_state: FacilitiesManagement::Procurement::SEARCH)
         @sent_offers = current_user.procurements.where(aasm_state: FacilitiesManagement::Procurement::SENT_OFFER)
         @in_draft = current_user.procurements.where(aasm_state: :DA_draft)
```
Comment (iceraluk) [inline app/controllers/facilities_management/beta/procurements_controller.rb]: you can do current_user.procurements.DA_draft to get all procurements in DA_draft state

File: db/schema.rb [hunk 2]
```diff
@@ -188,14 +190,15 @@
     t.index ["data"], name: "idx_fm_rate_cards_ginp", opclass: :jsonb_path_ops, using: :gin
   end
 
-  create_table "fm_rates", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+  create_table "fm_rates", id: false, force: :cascade do |t|
```
Comment (iceraluk) [inline db/schema.rb]: this should not change, make sure you match this with master. you might need to rebase with master and run: 
rake db:rollback STEP=6
rake db:migrate
this should clear your schema and show only the new changes that you added

File: app/controllers/facilities_management/beta/procurements_controller.rb (hunk 1 above)
Comment (ramchurnk) [reply app/controllers/facilities_management/beta/procurements_controller.rb]: Done.

File: db/schema.rb (hunk 2 above)
Comment (ramchurnk) [reply db/schema.rb]: done, thanks for the help :)

**Pull request merged**